src/client/pages/UnvalidatedEmailEmailSentPage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import useClientState from '@/client/lib/hooks/useClientState';
+import { buildQueryParamsString } from '@/shared/lib/queryParams';
+import { buildUrl } from '@/shared/lib/routeUtils';
+import { MainBodyText } from '../components/MainBodyText';
+import { EmailSent } from './EmailSent';
+
+interface Props {
+  noAccountInfo?: boolean;
+  formTrackingName?: string;
+}
+
+export const UnvalidatedEmailEmailSentPage = ({
+  noAccountInfo,
+  formTrackingName,
+}: Props) => {
+  const clientState = useClientState();
+  const {
+    pageData = {},
+    queryParams,
+    globalMessage = {},
+    recaptchaConfig,
+  } = clientState;
+  const { email } = pageData;
+  const { emailSentSuccess } = queryParams;
+  const { error } = globalMessage;
+  const { recaptchaSiteKey } = recaptchaConfig;
+
+  const queryString = buildQueryParamsString(queryParams, {
+    emailSentSuccess: true,
+  });
+
+  return (
+    <EmailSent
+      email={email}
+      changeEmailPage={buildUrl('/signin')}
+      resendEmailAction={buildUrl('/signin/email-sent/resend')}
+      queryString={queryString}
+      showSuccess={emailSentSuccess}
+      errorMessage={error}
+      noAccountInfo={noAccountInfo}
```

Comment:
It doesn't seem you're using `noAccountInfo`

So could this just be set to false or not included and be removed from the props?

```suggestion
```